riggerlib/__init__.py

```diff
@@ -258,18 +254,42 @@ def start_server(self):
         """
         Starts the TCP server if the ``server_enabled`` is True in the config.
         """
-        global _server
         self._server_hostname = self.config.get('server_address', '127.0.0.1')
         self._server_port = self.config.get('server_port', 21212)
         self._server_enable = self.config.get('server_enabled', False)
         if self._server_enable:
+            globt = threading.Thread(target=self.process_queue, name="global_queue_processor")
```

Comment:
These threads are required no matter what, and should be in ```__init__```. The server option existed if you wanted to fire hooks from an external process, you can of course run riggerlib locally and fire the hooks internally instead without going over zmq.